include/openenclave/attestation/custom_claims.h

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+#ifndef _OE_CUSTOM_CLAIMS
+#define _OE_CUSTOM_CLAIMS
+
+#include <openenclave/bits/evidence.h>
+#include <openenclave/bits/result.h>
+
+OE_EXTERNC_BEGIN
+#define OE_PLUGIN_CLAIMS_VERSION 1
+
+typedef struct _oe_plugin_claims_header
```

Comment:
If I understand this right, the term "custom" should be in the type name

Reply:
changed to oe_custom_claims_header